spec/models/custom_attribute_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe CustomAttribute do
+  it "return correct string field type" do
+    string_custom_attribute = FactoryGirl.build(:custom_attribute, :name => "foo", :value => "string", :resource_type => 'ExtManagementSystem')
```

Comment:
@lpichler any particular reason to do that? IMHO I like seeing them in the body of the test as they provide important context, I like to save `let`s for things that are not relevant but necessary for the setup